OpenRA.Game/FileFormats/Png.cs

```diff
@@ -350,13 +351,13 @@ public byte[] Save()
 
 				using (var data = new MemoryStream())
 				{
-					using (var compressed = new DeflaterOutputStream(data))
+					using (var compressed = new DeflaterOutputStream(data, new Deflater(Deflater.BEST_COMPRESSION)))
 					{
 						var rowStride = Width * PixelStride;
 						for (var y = 0; y < Height; y++)
 						{
-							// Write uncompressed scanlines for simplicity
-							compressed.WriteByte(0);
+							var filterType = (byte)0; // Assuming no filtering for simplicity
```

Comment:
Our convention is to put comments on their own line:
```suggestion
							// Assuming no filtering for simplicity
							var filterType = (byte)0;
```